src/components/breadcrumbs.js

```diff
@@ -0,0 +1,31 @@
+import { a, span } from 'react-hyperscript-helpers/lib/index'
+import { breadcrumb } from 'src/components/icons'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+
+
+export const breadcrumbElement = function(child, href) {
+  if (href) {
+    return a(
+      {
```

Comment:
Since we can't get our linter to do what we want, we may want to standardize on a style for these (I combined the open paren with the open brace in my PR, for example).